stablehlo/dialect/AssemblyFormat.cpp

```diff
@@ -286,12 +286,12 @@ void printDenseI64Array(OpAsmPrinter& p, Operation* op,
 
 ParseResult parseDenseI64Array(OpAsmParser& parser,
                                DenseIntElementsAttr& attr) {
-  DenseI64ArrayAttr arrayAttr =
-      DenseI64ArrayAttr::parse(parser, Type{}).dyn_cast<DenseI64ArrayAttr>();
```

Comment:
Let's perhaps use dyn_cast_or_null instead?

Reply:
Ah - that's how that function is used. Fixed.